lib/decidim/action_delegator/verification/engine.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    module Verification
+      # This is the engine that runs on the public interface of `ActionDelegator`.
+      class Engine < ::Rails::Engine
+        isolate_namespace Decidim::ActionDelegator::Verification
+
+        paths["db/migrate"] = nil
+        paths["lib/tasks"] = nil
```

Comment:
we should remove these two for the migrations and tasks to work, as we did in https://github.com/coopdevs/decidim-module-action_delegator/pull/2/, right?

Reply:
I think they should be nil here, the ActionDelegator::Engine will hold those migrations isn't? we only need one engine with the migrations